images/ansible/common/10_image_cleanup.yml

```diff
@@ -0,0 +1,54 @@
+- name: Gather current standard users
+  ansible.builtin.shell: >
+    cut -d: -f1,3 /etc/passwd | egrep ':[0-9]{4}$' | cut -d: -f1
+  register: _standard_users
+  
+- name: Remove all normal users except rhel and awx
+  ansible.builtin.user:
+    name: "{{ item }}"
+    state: absent
+    remove: true
+  loop: "{{ _standard_users.stdout_lines }}"
+  when: 
+    - item != "rhel"
+    - item != "awx"
+    
+- name: disable dnf automatic services
+  ansible.builtin.service:
+    name: "dnf-automatic.timer"
+    state: stopped
+
+- name: automatic.conf disable downloads
+  ansible.builtin.lineinfile:
+    path: /etc/dnf/automatic.conf
+    regexp: '^download_updates'
+    line: download_updates = no
+
+- name: automatic.conf disable updates
+  ansible.builtin.lineinfile:
+    path: /etc/dnf/automatic.conf
+    regexp: '^apply_updates'
+    line: apply_updates = no
+
+- name: Disable RHUI repos
```

Comment:
This may conflict with work that @monish-redhat was doing in the getting started with playbooks instruqt track. Can we confirm this task can still exist here or if can should be removed?